pkg/volume/rbd/rbd.go

```diff
@@ -362,15 +362,15 @@ func (r *rbdVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 	// create random image name
 	image := fmt.Sprintf("kubernetes-dynamic-pvc-%s", uuid.NewUUID())
 	r.rbdMounter.Image = image
-	rbd, sizeMB, err := r.manager.CreateImage(r)
+	rbd, sizeMB, err := r.manager.CreatePersistentVolumeImage(r)
 	if err != nil {
 		glog.Errorf("rbd: create volume failed, err: %v", err)
 		return nil, err
 	}
 	glog.Infof("successfully created rbd image %q", image)
 	pv := new(v1.PersistentVolume)
 	metav1.SetMetaDataAnnotation(&pv.ObjectMeta, volumehelper.VolumeDynamicallyCreatedByKey, "rbd-dynamic-provisioner")
-	rbd.SecretRef = new(v1.LocalObjectReference)
+	rbd.SecretRef = new(v1.SecretReference)
 	rbd.SecretRef.Name = secretName
```

Comment:
I expected to see `rbd.SecretRef.Namespace = secretNamespace` set from a `case "usersecretnamespace":` config param

Reply:
done